IPython/terminal/ptshell.py

```diff
@@ -223,13 +224,13 @@ def cursor_in_leading_ws(cli):
                                     & insert_mode
                                     & cursor_in_leading_ws
                                    ))
-        def _(event):
+        def _indent_buffer(event):
             event.current_buffer.insert_text(' ' * 4)
 
         # Pre-populate history from IPython's history database
         history = InMemoryHistory()
         last_cell = u""
-        for _, _, cell in self.history_manager.get_tail(self.history_load_length,
+        for __, ___, cell in self.history_manager.get_tail(self.history_load_length,
```

Comment:
My linter complain that each of those overwrite underscore defined
previously and which was unused.
On May 17, 2016 02:09, "Thomas Kluyver" <notifications@github.com> wrote:

> In IPython/terminal/ptshell.py
> <https://github.com/ipython/ipython/pull/9470#discussion_r63488140>:
>
> >              event.current_buffer.insert_text(' ' * 4)
> >
> >          # Pre-populate history from IPython's history database
> >          history = InMemoryHistory()
> >          last_cell = u""
> > -        for _, _, cell in self.history_manager.get_tail(self.history_load_length,
> > +        for __, ___, cell in self.history_manager.get_tail(self.history_load_length,
>
> ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ipython/ipython/pull/9470/files/12d304f48296a2566dbe711472707824baccd22e#r63488140>
>